display/tab.go

```diff
@@ -323,3 +323,15 @@ func (t *tab) applyAll() {
 		t.applyBottomBar()
 	}
 }
+
+// GetCurrentURI returns the currently selected URI
+func GetCurrentURI() string {
```

Comment:
Instead of making this a global function, make it a method of `tab` called `HighlightedURL`, and then call it like `tabs[curTab].HighlightedURL()`.

Reply:
Updated.